multi/stm32l4-multi/flash.c

```diff
@@ -317,6 +317,56 @@ void flash_getInfo(flashinfo_t *info)
 }
 
 
+ssize_t flash_writeRaw(uint32_t offset, const char *buff, size_t size)
+{
+	int ret;
+
+	if (program_isValidAddress(offset, size) == 0)
+		return 0;
+
+	mutexLock(flash_common.lock);
+	_program_unlock();
+
+	ret = _flash_wait();
+	if (ret != 0) {
+		return ret;
```

Comment:
Return, but flash_common.lock mutex is not unlocked and _program_lock is not called

Reply:
It's a bug copied from old code, I'll fix it everywhere, thank you

Reply:
Fixed, thanks